cypress/objects/common.js

```diff
@@ -1,7 +1,27 @@
 class Common {
-    checkIfUrlIncludes(text) {
-        cy.url().should('include', text);
+    checkIfUrlDoesNotInclude(text) {
+        cy.url().should('not.include', text);
     }
+
+    clickCertainButton(buttonType, text) {
+        cy.get(`[type="${buttonType}"]`).contains(text).click();
+    }
+
+    generateAUniqueString() {
+        return Date.now().toString();
+    }
+
+    visit(page) {
+        cy.visit(page);
+    }
```

Comment:
this function is redundant, it takes variable and pass it to another function, you could just call that function instead. you can have visit('/tasks') action in taskPage.js for visit action